### ISSUE TITLE
Disabling PAN does not seem to be working

### ISSUE THREAD

**Issue by hyoun98** [mikefrederick/MFSideMenu#76]
On certain viewcontrollers within a particular tab, I do not wish to enable the sidemenu and panning.   I attempted to turn it off by doing the following.  However, I am able to swipe left or right.  How do i disable the sidemenu?  Am I using the right technique?  Thank you
 UINavigationController *navController = [[UINavigationController alloc] initWithRootViewController:masterViewController];

 [navController.sideMenu setPanMode:MFSideMenuPanModeNone];

**Comment by mikefrederick** [comment]
Have you initialized a sideMenu with your navigationController? Is navController.sideMenu nil?

I just double checked and this works fine:

``` objective-c
MFSideMenu *sideMenu = [MFSideMenu menuWithNavigationController:masterNavigationController
                                             leftSideMenuController:leftSideMenuController
                                            rightSideMenuController:rightSideMenuController];
sideMenu.panMode = MFSideMenuPanModeNone;
```

**Comment by hyoun98** [comment]
Yes, I did initialize the sideMenu as your example shows.  Is it possible to have a two different sideMenu's (i.e. one for each tab ) where the panMode is ON for the first tab, but OFF for the second tab.  I tried to do that, but it seemed  that both tabs carried the same behavior.  

Additionally, I also noticed if i have one tab where i am using the sideMenu (with panMode enabled) and a second tab with no side menu, but just a navigation controller, the user can swipe the controller in the second tab as if there was a side menu to the left of the right.   Is there any way to prevent this from happening?  Shall I create a different issue for this?  Many thanks.
